doc/howtos/ip-address-config.rst

```diff
@@ -0,0 +1,119 @@
+.. _ip-address-config:
+
+Configuring an IP Address in the Ostro |trade| OS
+######################################################
+
+The Ostro Project uses Connection Manager (ConnMan), a Linux daemon and command-line utility, for managing internet connections 
+within embedded devices.
+
+ConnMan supports these network technologies:
+
+* Ethernet, using a built-in card, connected via host or client side USB
+* WiFi, using wpa_supplicant
+* Celluar connectivity using oFono
+* Bluetooth connectivity using BlueZ
+
+By default all technologies are disabled at the very first startup
+to prevent unwanted wireless or wired communication from happening. ConnMan remembers the state of network
+technologies over reboots. 
+(You can read more about ConnMan at https://01.org/connman/documentation.)
+
+This tech note explains how to connect your device to a network manually using the ``conmannctl`` command line client.  For proper integration in 
+an application, the D-Bus service API (and its bindings) should be used.  
```

Comment:
A couple of nitpicks:
- Shouldn't command line be spelled with an hyphen, i.e. command-line?
- Technically speaking, we'd expect people to use more than just ConnMan service API for proper integration, they'd also use its Manager API, Session API, etc. so I'd suggest we just say "... the D-Bus API (and its bindings) should be used."